### PR TITLE
Clouds: Make cloud area radius settable in .conf

### PR DESCRIPTION
**Pull request by paramat**
See issue #2662 for details and comments from players. Default is set to the current value of 12.

### CODE REVIEW

File: src/clouds.cpp [hunk 1]
```diff
@@ -99,7 +99,7 @@ void Clouds::render()
 		Clouds move from X+ towards X-
 	*/
 
-	const s16 cloud_radius_i = 12;
+	const s16 cloud_radius_i = g_settings->getS16("cloud_radius");
```
Comment (est31) [inline src/clouds.cpp]: Why not make it unsigned?

Comment (paramat) [inline src/clouds.cpp]: I'm just staying consistent with the current code which uses s16.

Comment (est31) [inline src/clouds.cpp]: I don't think it should have been u16.

**Comment by est31** [comment]
Could you document it in `minetest.conf.example`?

**Comment by paramat** [comment]
Yes will do.

**Comment by est31** [comment]
9345ba7b6a6cd19e0af0f51be1154ae571332f04

**Comment by nerzhul** [comment]
Do you look at the render performance regression ? Using a cached setting is better than calling settings everytime

**Comment by est31** [comment]
That's what I've done in my modifications.

**Pull request closed without merging**